src/legacy/ui/public/chrome/directives/header_global_nav/components/header.tsx

```diff
@@ -88,10 +89,25 @@ function extendRecentlyAccessedHistoryItem(
   const href = relativeToAbsolute(chrome.addBasePath(recentlyAccessed.link));
   const navLink = navLinks.find(nl => href.startsWith(nl.subUrlBase));
 
+  let titleAndAriaLabel = recentlyAccessed.label;
+  if (navLink) {
+    const objectTypeForAriaAppendix = navLink ? navLink.title : '';
```

Comment:
see previous line. `navLink ?` looks like extra check

Reply:
@alexwizp totally agreed, thank you